package.json

```diff
@@ -26,7 +26,6 @@
     "test:coverage": "npm run test:unit -- --coverage",
     "test:watch": "tsc-watch --onSuccess 'npm run test:unit'",
     "start": "bin/start",
-    "prepare": "npm run build",
```

Comment:
And why remove the prepare step?

Reply:
Because it's not used correct :) `prepare` is ran after `npm install` and it would fail, cos `tsconfig.json` is not inside. Before Node 16 it looks like was just a WARN and not error